sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaMetrics.java

```diff
@@ -37,6 +37,7 @@ public interface KafkaMetrics {
 
   void updateBacklogBytes(String topic, int partitionId, long backlog);
 
+  /*Used to update all metrics in container*/
```

Comment:
how about
Flushes the buffered metrics to the current metric container for this thread.

could we rename it to flushBufferedMetrics?

Reply:
Done